src/main/java/sirius/kernel/commons/TimeProvider.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Made with all the love in the world
+ * by scireum in Remshalden, Germany
+ *
+ * Copyright by scireum GmbH
+ * http://www.scireum.de - info@scireum.de
+ */
+
+package sirius.kernel.commons;
+
+import sirius.kernel.di.std.Register;
+
+import java.time.Instant;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+/**
+ * Provides wrapper for static time functions which can be injected using a {@link sirius.kernel.di.std.Part}
+ * annotation.
+ * <p>
+ * Using this wrapper instead of the static methods permits to use mocking in tests.
+ */
+@Register(classes = TimeProvider.class)
+public class TimeProvider {
+    public LocalDateTime localDateTimeNow() {
```

Comment:
bei jeder methode noch n kurzen javadoc ala "Wrapper for {@link...LocalDateTime.now}" und einem kurzen @return ala "Returns the current date as LocalDate" oder so - sonst meckert checkstyle^^